nemo/collections/nlp/modules/common/token_classifier.py

```diff
@@ -70,6 +70,88 @@ def __init__(
             use_transformer_init: whether to initialize the weights of the classifier head with the same approach used in Transformer
         """
         super().__init__()
+        self.mlp = MultiLayerPerceptron(
+            hidden_size, num_classes, num_layers=num_layers, activation=activation, log_softmax=log_softmax
+        )
+        self.dropout = nn.Dropout(dropout)
+        if use_transformer_init:
+            self.apply(lambda module: transformer_weights_init(module, xavier=False))
+
+    @typecheck()
+    def forward(self, hidden_states):
+        """
+        Performs the forward step of the module.
+        Args:
+            :param hidden_states: batch of hidden states (for example, from the BERT encoder module)
+                [BATCH_SIZE x SEQ_LENGTH x HIDDEN_SIZE]
+            :return: logits value for each class [BATCH_SIZE x SEQ_LENGTH x NUM_CLASSES]
+        """
+        hidden_states = self.dropout(hidden_states)
+        logits = self.mlp(hidden_states)
+        return logits
+
+    def save_to(self, save_path: str):
+        """
+        Saves the module to the specified path.
+        Args:
+            :param save_path: Path to where to save the module.
+        """
+        pass
+
+    @classmethod
+    def restore_from(cls, restore_path: str):
+        """
+        Restores the module from the specified path.
+        Args:
+            :param restore_path: Path to restore the module from.
+        """
```

Comment:
:param and :return I could be the who added them actually, would appreciate if you delete them

Reply:
done